config/plugins/tours/core.history.yaml

```diff
@@ -122,11 +127,11 @@ steps:
       position: "left"
       textinsert: "WWFSMD"
       preclick:
-        -  "#current-history-panel > div.controls > div.search > div > input"
+        - "#current-history-panel > div.controls > div.search > div > input"
 
     - element: "#history-options-button"
       title: "History Options"
-      intro: "In the History menue you will find a lot more usful History options."
+      intro: "In the History menue you will find a lot more useful History options."
       position: "left"
       postclick:
```

Comment:
@guerler @dannon during this step something wired happened. Would be wonderful if you can have a look at it.
We can also remove this last step.